openhands_cli/agent_chat.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python3
+"""
+Simple agent chat functionality for OpenHands CLI.
+Provides a basic conversation interface with an AI agent.
+"""
+
+import os
+import sys
+from typing import Optional
+
+# Ensure we use the agent-sdk openhands package, not the main OpenHands package
+# Remove the main OpenHands code path if it exists
+if '/openhands/code' in sys.path:
+    sys.path.remove('/openhands/code')
+
+from pydantic import SecretStr
+from prompt_toolkit import PromptSession, print_formatted_text
+from prompt_toolkit.formatted_text import HTML
+from prompt_toolkit.shortcuts import clear
+
+try:
+    from openhands.core import (
+        LLM,
+        CodeActAgent,
+        Conversation,
+        EventType,
+        LLMConfig,
+        LLMConvertibleEvent,
+        Message,
+        TextContent,
+        Tool,
+        get_logger,
+    )
+    from openhands.tools import (
+        BashExecutor,
+        FileEditorExecutor,
+        execute_bash_tool,
+        str_replace_editor_tool,
+    )
+except ImportError as e:
+    print_formatted_text(HTML(f"<red>Error importing agent SDK: {e}</red>"))
+    print_formatted_text(HTML("<yellow>Please ensure the openhands-sdk is properly installed.</yellow>"))
+    sys.exit(1)
+
+
+logger = get_logger(__name__)
+
+
+class SimpleAgentChat:
```

Comment:
@openhands please make sure that we're porting over the confirmation mode functionality from CLI in OpenHands repo 

keep it as simple as possible; everytime the agent wants to do anything it should ask for approval from the user